assets/js/modules/analytics/components/settings/GA4SettingsControls.js

```diff
@@ -244,21 +244,27 @@ export default function GA4SettingsControls( {
 						outlined
 					>
 						<Option value={ matchedProperty?._id || '' }>
-							{ ! matchedProperty?._id ||
-							! matchedProperty?.displayName
-								? ''
-								: sprintf(
-										/* translators: 1: Property name. 2: Property ID. */
-										_x(
-											'%1$s (%2$s)',
-											'Analytics property name and ID',
-											'google-site-kit'
-										),
-										matchedProperty.displayName,
-										measurementIDs?.[
-											matchedProperty._id
-										] || ''
-								  ) }
+							{ matchedProperty?._id &&
+							matchedProperty?.displayName ? (
+								<Fragment>
+									{ measurementIDs?.[ matchedProperty._id ]
+										? sprintf(
```

Comment:
This is getting a bit hard to read. Can we simplify it a bit to avoid so many conditions? I'm referring to the whole inner content of `Option` here, I just can't select it all 😄

Reply:
I've updated this to be a little more readable. Thank you!